docs/tutorials/adding-a-module-to-your-runtime.md

```diff
@@ -172,83 +172,111 @@ Now that we have successfully imported the Contracts module crate, we need to ad
 
 ```rust
-// From the reference documentation, also found in `contracts` module:
+// From the reference documentation, also found in `contracts` pallet:
 //   https://github.com/paritytech/substrate/blob/master/palette/contracts/src/lib.rs
 
 type Currency: Currency<Self::AccountId>
 ```
 
-Fortunately, the Balances module implements this type, so we can simply reference `Balances` to gain access to it.
+Fortunately, the Balances pallet implements this type, so we can simply reference `Balances` to gain access to it.
 
-Similarly, `type DetermineContractAddress` requires the trait `ContractAddressFor`. The Contracts module itself implements a type with this trait in `contract::SimpleAddressDeterminator`, thus we can use that implementation to satisfy our `contracts::Trait`. At this point, I really recommend you explore the source code of the [Contracts module](https://github.com/paritytech/substrate/blob/master/palette/contract/src/lib.rs) if things don't make sense or you want to gain a deeper understanding.
+Similarly, `type DetermineContractAddress` requires the trait `ContractAddressFor`. The Contracts pallet itself implements a type with this trait in `contract::SimpleAddressDeterminator`, thus we can use that implementation to satisfy our `contracts::Trait`. At this point, I really recommend you explore the source code of the [Contracts pallet](https://github.com/paritytech/substrate/blob/master/frame/contracts/src/lib.rs) if things don't make sense or you want to gain a deeper understanding.
 
 ### Adding Contract to the Construct Runtime Macro
 
-Next, we need to add the module to the `construct_runtime!` macro. For this, we need to determine the types that the module exposes so that we can tell the our runtime that they exist. The complete list of possible types can be found in the [`construct_runtime!` macro documentation](https://substrate.dev/rustdocs/master/palette_support/macro.construct_runtime.html).
+Next, we need to add the pallet to the `construct_runtime!` macro. For this, we need to determine the types that the pallet exposes so that we can tell the our runtime that they exist. The complete list of possible types can be found in the [`construct_runtime!` macro documentation](https://substrate.dev/rustdocs/master/palette_support/macro.construct_runtime.html).
 
-If we look at the Contracts module in detail, we know it has:
+If we look at the Contracts pallet in detail, we know it has:
 
 * Module **Storage**: Because it uses the `decl_storage!` macro.
 * Module **Event**s: Because it uses the `decl_event!` macro.
 * **Call**able Functions: Because it has dispatchable functions in the `decl_module!` macro.
 * **Config**uration Values: Because the `decl_storage!` macro has `config()` parameters.
-* The **Module** type from the `decl_module!` macro.
+* The **pallet** type from the `decl_pallet!` macro.
```

Comment:
Right now every pallet has `decl_module!` macro instead of the `decl_pallet!` macro so I am assuming this will be future changes as the master branch doesn't reflect `decl_pallet!` right now but it is a schedule change.

Reply:
Or it is a mistake to have `decl_pallet!`, `It should be replaced with decl_module!`

Reply:
Nice! you are right, thanks for the catch.